src/main/java/org/wiremock/webhooks/Webhooks.java

```diff
@@ -80,8 +84,12 @@ private void triggerWebhook(ServeEvent serveEvent, Parameters parameters) {
       }
       definition = applyTemplating(definition, serveEvent);
       request = buildRequest(definition);
+
+      serveEvent.appendSubEvent(SubEvent.INFO, LoggedRequest.createFrom(request));
     } catch (Exception e) {
-      notifier().error("Exception thrown while configuring webhook", e);
+      final String msg = "Exception thrown while configuring webhook";
+      notifier().error(msg, e);
```

Comment:
Suggest we don't add a notifier() call here since we're intending to phase this out and instead log the current request's sub-events instead.

Reply:
After further discussion we decided to leave the notifer in place as it was there prior to this PR